torchvision/io/__init__.py

```diff
@@ -126,10 +127,10 @@ def __next__(self):
             raise StopIteration
         return {"data": frame, "pts": pts}
 
-    def __iter__(self):
+    def __iter__(self) -> Iterator['VideoReader']:
         return self
 
-    def seek(self, time_s: float):
+    def seek(self, time_s: float) -> 'VideoReader':
```

Comment:
Could be replaced with `VideoReader` once we drop Python 3.6 support. See [here](https://stackoverflow.com/a/33533514).

Reply:
Nice. 
I learn something new everyday because of review from torchvision maintainers 😇

There are couple of more places where I have used this workaround in typing in torchvision. I will keep a note of this point.

Reply:
@datumbox yup fully agree! But for now (this PR), I guess we keep the string version?

Reply:
yes that's right, nothing we can do for now.